packages/modules/promotion/src/models/promotion.ts

```diff
@@ -51,7 +51,6 @@ export default class Promotion {
     fieldName: "campaign_id",
     nullable: true,
     mapToPk: true,
-    onDelete: "set null",
```

Comment:
Was this the culprit?

Reply:
no, this just wasn't needed. The issue is still a mystery. 😄